lib/NUPACK/emscripten/FullEnsemble.cpp

```diff
@@ -212,4 +216,92 @@ FullAdvancedResult* FullEnsembleNoBindingSite(const std::string& seqString, int
     return result;
 }
 
+//this is the current implementation of generating the dot paren structure from the fold as of 7/20/22
+//this seams to be lacking compaired to nupacks PrintDNAStructure called by the basics apps
+//when displaying the structure
+std::string getDotParens_OldMethod(bool pseudoknotted, const int seqlength, oneDnaStruct *currentStruct) {
```

Comment:
* I see the separation you're trying to make, but please keep it camelcase all the way through
* Just call it getDotParens - we don't have an alternate implementation right now